### DISCUSSION TITLE
New test release mu-argus

### DISCUSSION THREAD

**Discussion by ppdewolf** [sdcTools/UserSupport#238] [Team Testers]
Dear testers,

during the last User Group Meeting in February 2022, you volunteered to be a tester of SDC tools.

Now is the time for your first contribution!

I have just uploaded a new version of mu-argus ([version 5.1.7](https://github.com/sdcTools/muargus/releases/tag/v5.1.7)) to be tested. The main addition is that now multiple similarity profiles are possible for Targeted Record Swapping.

So I would appreciate your feedback on this new feature. And your check on whether the "old" features are still functioning as they should ;-)

Even though the summer holidays are near, I would appreciate your feedback before 1 August 2022. After that I will make the release official.

Thanks!
Peter-Paul

**Comment by ppdewolf** [comment]
NB: the test release can be found here: [v5.1.7](https://github.com/sdcTools/muargus/releases/tag/v5.1.7).

I incorrectly included the source of the master branch (i.s.o. the TargetedRecordSwapping branch) that resulted in an incorrect timestamp. As a consequence this pre-release is way down the list of releases.

**Comment by tdlinehancso** [comment]
Peter,

Here is the testing feedback on uArgus 5.1.7b3 from Karina Dineen and Tim Linehan of the Central Statistics Office
Testing is based on a synthetic dataset we use. It's available upon request if required.

(See attached file: CSO Ireland Feedback for MuArgus 5.1.7b3.docx)

Regards,

Tim

"Peter-Paul de Wolf" ---11/07/2022 14:56:17---CAUTION: This email originated from outside of the CSO. Do not click links or open   attachments unl

From:	"Peter-Paul de Wolf" ***@***.***>
To:	"sdcTools/testers" ***@***.***>
Cc:	"Subscribed" ***@***.***>
Date:	11/07/2022 14:56
Subject:	[sdcTools/testers] New test release mu-argus (#2)





CAUTION: This email originated from outside of the CSO. Do not click links or open attachments unless you recognise the sender and know the content is safe. If in doubt contact the Service Desk at Extension 5666.





Dear testers,
during the last User Group Meeting in February 2022, you volunteered to be a tester of SDC tools.
Now is the time for your first contribution!
I have just uploaded a new version of mu-argus (version 5.1.7) to be tested. The main addition is that now multiple similarity profiles are possible for Targeted Record Swapping.
So I would appreciate your feedback on this new feature. And your check on whether the "old" features are still functioning as they should ;-)
Even though the summer holidays are near, I would appreciate your feedback before 1 August 2022. After that I will make the release official.
Thanks!
Peter-Paul

**Comment by tdlinehancso** [comment]
Thanks a lot! I’ll have a look later.

Van: ***@***.*** ***@***.***>
Verzonden: vrijdag 29 juli 2022 15:41
Aan: sdcTools/testers ***@***.***>
CC: ***@***.***; Wolf, P.P. de (Peter-Paul) ***@***.***>
Onderwerp: [*Nieuwsbrief*] Re: [sdcTools/testers] New test release mu-argus (#2)


Peter,

Here is the testing feedback on uArgus 5.1.7b3 from Karina Dineen and Tim Linehan of the Central Statistics Office
Testing is based on a synthetic dataset we use. It's available upon request if required.

(See attached file: CSO Ireland Feedback for MuArgus 5.1.7b3.docx)

Regards,

Tim

[Inactive hide details for "Peter-Paul de Wolf" ---11/07/2022 14:56:17---CAUTION: This email originated from outside of the CSO.]"Peter-Paul de Wolf" ---11/07/2022 14:56:17---CAUTION: This email originated from outside of the CSO. Do not click links or open   attachments unl

From: "Peter-Paul de Wolf" ***@***.******@***.***>>
To: "sdcTools/testers" ***@***.******@***.***>>
Cc: "Subscribed" ***@***.******@***.***>>
Date: 11/07/2022 14:56
Subject: [sdcTools/testers] New test release mu-argus (#2)

________________________________

CAUTION: This email originated from outside of the CSO. Do not click links or open attachments unless you recognise the sender and know the content is safe. If in doubt contact the Service Desk at Extension 5666.



Dear testers,

during the last User Group Meeting in February 2022, you volunteered to be a tester of SDC tools.

Now is the time for your first contribution!

I have just uploaded a new version of mu-argus (version 5.1.7) to be tested. The main addition is that now multiple similarity profiles are possible for Targeted Record Swapping.

So I would appreciate your feedback on this new feature. And your check on whether the "old" features are still functioning as they should ;-)

Even though the summer holidays are near, I would appreciate your feedback before 1 August 2022. After that I will make the release official.

Thanks!
Peter-Paul
